assets/css/edd-admin.css

```diff
@@ -1797,10 +1776,6 @@ input[class*="edd-price-field"] {
 .edd_repeatable_product_wrapper td {
 	overflow: visible;
 }
-.edd-order-details-label-mobile {
```

Comment:
There are a couple more instances of this class elsewhere  in this file that are no longer needed.

Reply:
I think I got them all!